### DISCUSSION TITLE
How to search for Files with Filename(without providing text inside file)

### DISCUSSION THREAD

**Discussion by sailukar** [dnGrep/dnGrep#1059] [Q&A] [answered]
How to search for Files with Filename(without providing text inside file).

I would like to search for files with filenames as '1.txt' and 'filenameWithoutExtension' in a folder C:\Folder using dngrep

**Comment by doug24** [comment, marked as answer]
First, in the dnGrep Options dialog, scroll down to the 'Seach results' section, and check the option "List files using the file name pattern when the 'Search for' field is empty".  Then on the main window, leave the 'Search for' field empty.

If you are looking for files matching '1.txt' and 'filenameWithoutExtension' exactly, set the pattern type to 'Wildcard' at the top of the 'More...' filters panel, and put the file name in the 'Patterns to match' field.   You can do them one at a time or both together like this: `1.txt;filenameWithoutExtension`

If you are looking for all files with no extension, change the pattern type to 'Regular expression' at the top of the 'More...' filters panel. then put this regular expression in the 'Patterns to match' field:  `^[^.]*$`

**Reply by sailukar** [reply]
Appreciate quick response, this is the solution iam looking for.
